.github/workflows/build.yml

```diff
@@ -23,14 +23,14 @@ jobs:
 
     - name: Install OBS deps
       run: |
-        brew install FFmpeg x264 Qt5 cmake mbedtls swig
+        brew bundle --file=obs-mac-virtualcam/Brewfile
```

Comment:
Neat I didn't know about this. Definitely nice to not have to duplicate this line between the two workflows.